misocho/questioner
# #163177066 Refactors questions and meetups endpoints

#### What does this PR do?
Refactors questions and meetups endpoints 
#### Description of Task to be completed?
* Refactor questions and meetups endpoints
* Add tests to increase coverage 
#### How should this be manually tested?
* Clone the repo
* Navigate to project directory
* Create and activate a virtual environment
* pip install requirements.txt
* Create a branch and pull from ch-api-refractor-163177066
* Check code
* Run pytest to test endpoints
* Run endpoints on postman
#### What are the relevant pivotal tracker stories?
#163177066
https://www.pivotaltracker.com/n/projects/2235306
